server/src/routes/api/v1/roomRouter.js

```diff
@@ -0,0 +1,14 @@
+import express from "express"
+import v4 from "uuid"
+
+const roomRouter = new express.Router()
+
+roomRouter.post("/new", (req, res) => {
+  try {
+    return res.status(201).json({ roomId: v4() })
```

Comment:
you'll probably be persisting rooms in the future, so this may change to the actual id of a room at some point